dmapiclient/data.py

```diff
@@ -330,6 +330,11 @@ def export_users(self, framework_slug):
             "/users/export/{}".format(framework_slug)
         )
 
+    def email_address_has_valid_buyer_domain(self, email_address):
```

Comment:
We talked about naming these `is_something_something` so that it's immediately obvious which methods should return booleans.